pkg/argocd/argocd.go

```diff
@@ -183,11 +183,17 @@ func FilterApplicationsForUpdate(apps []v1alpha1.Application, patterns []string)
 	return appsForUpdate, nil
 }
 
-func parseImageList(updateImage string) *image.ContainerImageList {
-	splits := strings.Split(updateImage, ",")
-	results := make(image.ContainerImageList, len(splits))
-	for i, s := range splits {
-		results[i] = image.NewFromIdentifier(strings.TrimSpace(s))
+func parseImageList(annotations map[string]string) *image.ContainerImageList {
+	results := make(image.ContainerImageList, 0)
+	if updateImage, ok := annotations[common.ImageUpdaterAnnotation]; ok {
+		splits := strings.Split(updateImage, ",")
+		for _, s := range splits {
+			img := image.NewFromIdentifier(strings.TrimSpace(s))
+			if kustomizeImage, ok := annotations[fmt.Sprintf(common.KustomizeApplicationNameAnnotation, img.ImageAlias)]; ok {
```

Comment:
Can you please use `image.GetParameterKustomizeImageName()` instead of going through annotations manually, please?